modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -314,6 +314,20 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_control(
+			'n_accordion_animation_duration',
+			[
+				'label' => esc_html__( 'Animation Duration', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 's', 'ms' ],
```

Comment:
```suggestion
				'size_units' => [ 's', 'ms', 'custom' ],
```

Reply:
@rodolphebertozzo 
Thank's for noticing.
Great suggestion, This was the original product manager's suggestion.
In this case, the toggle animation is controlled by the Javascript animate API.
The animate API receives data in milliseconds.
Just for curiosity, which other time units a user would like to use?
Minutes? Hours?
ודא

Reply:
we have only `s` and `ms`, other using are css functions that return calculations of `s` and `ms`.

Reply:
@rami-elementor so should we add custom units to this control?